doc/api/inspector.md

```diff
@@ -180,8 +182,32 @@ session.post('Profiler.enable', () => {
 });
 ```
 
+### Heap Profiler
+
+Here's a simple example showing how to use the [heap profiler][]:
+
+```js
+const inspector = require('inspector');
+const fs = require('fs');
+const session = new inspector.Session();
+
+session.connect();
+
+fs.writeFileSync('profile.heapsnapshot', '');
+
+session.on('HeapProfiler.addHeapSnapshotChunk', function(m) {
+  fs.appendFileSync('profile.heapsnapshot', m.params.chunk);
```

Comment:
I’d rather open a file descriptor and `fs.writeSync` to it, as a best practice for avoiding race conditions?